core/dbt/events/functions.py

```diff
@@ -0,0 +1,213 @@
+
+from colorama import Style
+import dbt.events.functions as this  # don't worry I hate it too.
+from dbt.events.types import Cli, Event, File, ShowException
+import dbt.flags as flags
+# TODO this will need to move eventually
+from dbt.logger import SECRET_ENV_PREFIX, make_log_dir_if_missing
+import json
+import logging
+from logging import Logger
+from logging.handlers import WatchedFileHandler
+import os
+from typing import List
+
+
+# create the global file logger with no configuration
+global FILE_LOG
+FILE_LOG = logging.getLogger('default_file')
+
+# set up logger to go to stdout with defaults
+# setup_event_logger will be called once args have been parsed
+global STDOUT_LOG
+STDOUT_LOG = logging.getLogger('default_stdout')
+STDOUT_LOG.setLevel(logging.INFO)
+stdout_handler = logging.StreamHandler()
+stdout_handler.setLevel(logging.INFO)
+STDOUT_LOG.addHandler(stdout_handler)
+global color
+format_color = True
+global json
+format_json = False
```

Comment:
I don't understand why global is needed here. `global json` is referring to the json module here I think as well.

Reply:
oh good catch. I was supposed to change that to global format_json, but it's working without that declaration. so I guess I don't need it.